docs/documentation/en/sql-reference/sql-statements/Data Definition/create-function_EN.md

```diff
@@ -49,23 +53,33 @@ If the `function_name'contains the database name, the custom function will be cr
 
 1. Create a custom scalar function
 
-```
-CREATE FUNCTION my_add(INT, INT) RETURNS INT PROPERTIES (
-"Symbol"=""\\\\\\\\ zn9doris\\\ udf6addudfepns\\ FunctionContexterkns\\ INTVales 4\,
-"object file" ="http://host:port /libmyadd.so"
-);
-```
+	```
+	CREATE FUNCTION my_add(INT, INT) RETURNS INT PROPERTIES (
+	"Symbol"=""\\\\\\\\ zn9doris\\\ udf6addudfepns\\ FunctionContexterkns\\ INTVales 4\,
+	"object file" ="http://host:port /libmyadd.so"
+	);
+	```
+2. Create a custom scalar function with prepare/close functions
 
-2. Create a custom aggregation function
+	```
+	CREATE FUNCTION my_add(INT, INT) RETURNS INT PROPERTIES (
+   		"symbol" = 	"_ZN9doris_udf6AddUdfEPNS_15FunctionContextERKNS_6IntValES4_",
+   		"prepareFnSymbol" = "_ZN9doris_udf14AddUdf_prepareEPNS_15FunctionContextENS0_18FunctionStateScopeE",
+   		"closeFnSymbol" = "_ZN9doris_udf12AddUdf_closeEPNS_15FunctionContextENS0_18FunctionStateScopeE",
+    	"object_file" = "http://host:port/libmyadd.so"
+	);
+	```
 
-```
-CREATE AGGREGATE FUNCTION my_count (BIGINT) RETURNS BIGINT PROPERTIES (
-"init u fn"= "ZN9doris, udf9CountInitEPNS -u 15FunctionContextEPNS, u 9BigIntValE",
-"Update  fn" = " zn9doris \ udf11Countupdateepns \ \ FunctionContexterkns \ Intvalepns  bigintvale",
-"Merge fn"="\ zn9doris\\ udf10CountMergeepns\ \ FunctionContexterkns\ Bigintvaleps2\\\\\\\\\\\\\
-"Finalize \ fn" = "\ zn9doris \ udf13Count Finalizepns \\ FunctionContexterkns \ Bigintvale",
-"object" file ="http://host:port /libudasample.so"
-);
-```
+3. Create a custom aggregation function
+	
+	```
+	CREATE AGGREGATE FUNCTION my_count (BIGINT) RETURNS BIGINT PROPERTIES (
+	"init u fn"= "ZN9doris, udf9CountInitEPNS -u 15FunctionContextEPNS, u 9BigIntValE",
+	"Update  fn" = " zn9doris \ udf11Countupdateepns \ \ FunctionContexterkns \ Intvalepns  bigintvale",
```

Comment:
these backslash may be generated by Baidu translator, can be removed.

Reply:
done